update-log/v3.0.0.md

```diff
@@ -0,0 +1,98 @@
+# Solana Exporter v3.0.0
+
+## Key Changes
+
+The new `solana-exporter` (renamed from `solana_exporter`) contains many new metrics, standardised naming conventions 
+and more configurability.
+
+## What's Changed
+### Metric Updates
+#### New Metrics
+
+Below is a list of newly added metrics (see the [README](https://github.com/asymmetric-research/solana-exporter/blob/master/README.md) 
+for metric descriptions):
+
+* `solana_account_balance` (<u><strong>[@johnstonematt](https://github.com/johnstonematt)</strong></u>)
+* `solana_node_is_healthy` (<u><strong>[@GranderStark](https://github.com/GranderStark)</strong></u>)
+* `solana_nude_num_slots_behind` (<u><strong>[@GranderStark](https://github.com/GranderStark)</strong></u>)
+* `solana_node_minimum_ledger_slot` (<u><strong>[@GranderStark](https://github.com/GranderStark)</strong></u>)
+* `solana_node_first_available_block` (<u><strong>[@GranderStark](https://github.com/GranderStark)</strong></u>)
+* `solana_cluster_slots_by_epoch_total` (<u><strong>[@johnstonematt](https://github.com/johnstonematt)</strong></u>) 
+* `solana_validator_fee_rewards` (<u><strong>[@johnstonematt](https://github.com/johnstonematt)</strong></u>)
+* `solana_validator_block_size` (<u><strong>[@johnstonematt](https://github.com/johnstonematt)</strong></u>)
+* `solana_node_block_height` (<u><strong>[@GranderStark](https://github.com/GranderStark)</strong></u>)
```

Comment:
Did you miss the `solana_block_size` metric from #46? I think that one warrants a blurb at the top that mentions it allowing monitoring and alerting on blocks with no user transactions (vote only blocks are very bad for the network obviously).

Reply:
`solana_validator_block_size` is there